docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+web:
+    build: .
+    volumes:
+        - .:/app
```

Comment:
Docker use feature only avialable on linux (cgroups, namespace, etc...). MacOs and Windows users have to run a VirtualMachine and launch docker inside it (it's Boot2Docker)
When you mount a volume `.:/app`, boot2docker create a sharedFolder by using vboxfs (or NFS with some tricks) wich is really slow.
But you have no choice, you have to share the sources between the Host and the container. Except for files you don't care (.gitignore is a good entrypoint to find this files).

By adding new volume `/app/app/cache` (notice there isn't any `:` => it's not a folder shared between the host and the container), docker will create a new volume for the container (it's the same thing when you use volume's container and `volumes_from`). This volume is not shared with the host, it does not use vboxsf (or NFS) and is much more faster.